src/selectors/layout.js

```diff
@@ -68,35 +68,64 @@ export const getGraphSize = createSelector(
   graph => graph.graph()
 );
 
+/**
+ * Return the displayed width of the sidebar
+ */
+export const getSidebarWidth = (visibleSidebar, outerChartWidth) => {
+  const defaultSidebarWidth = 300; // from _variables.scss
+  const breakpointSmall = 480; // from _variables.scss
+  if (visibleSidebar && outerChartWidth > breakpointSmall) {
+    return defaultSidebarWidth;
+  }
+  return 0;
+};
+
+/**
+ * Convert the DOMRect into an Object, mutate some of the properties,
+ * and add some useful new ones
+ */
+export const getChartSize = createSelector(
+  [getVisibleSidebar, state => state.chartSize],
+  (visibleSidebar, chartSize) => {
+    const { left, top, width, height } = chartSize;
+    if (!width || !height) {
+      return {};
+    }
+    const sidebarWidth = getSidebarWidth(visibleSidebar, width);
+    return {
+      left,
+      top,
+      outerWidth: width,
+      outerHeight: height,
+      width: width ? width - sidebarWidth : width,
```

Comment:
This should be `sidebarWidth ? width - sidebarWidth : width` (probably)

Reply:
ahhhh this is a leftover relic from before I added the check for `if (!width || !height) {` on line 91. I can now remove the ternary check completely.